engine/src/main/java/org/terasology/config/flexible/FlexibleConfig.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2016 MovingBlocks
```

Comment:
That didn't make any difference whatsoever to the copyright template. In fact, I think it has broken my project. I may have to re-clone it.